src/fontra/views/editor/editor.html

```diff
@@ -5,6 +5,13 @@
     <title>Fontra</title>
     <link href="/css/core.css" rel="stylesheet" />
     <link href="/editor/editor.css" rel="stylesheet" />
+    <script type="importmap">
```

Comment:
I thought you discovered importmap is redundant. In your remote demo you still do
```js
import { objectsEqual } from "fontra/core/utils.js";
```
So it will work if you would do this instead:
```
import { objectsEqual } from "/core/utils.js";
```
Right? If so, let's remove the importmap.

Reply:
Yes, ok. I was unsure if we treat it as an alias, to make it clear we import something from fontra. It could be helpful if the plugins are going to have external packages in future.

But I delete it. Better not to have an alias if it's importable by without import map (it's), to avoid possible confusion. Packaging is always confusing.